stable/cockroachdb/templates/cockroachdb-statefulset.yaml

```diff
@@ -249,7 +249,9 @@ spec:
           periodSeconds: 5
           failureThreshold: 2
         resources:
-{{ toYaml .Values.resources | indent 10 }}
+          requests:
```

Comment:
Actually, we always suggest to not specify defaults for resources. I'd rather fix it the other way around and comment the defaults out.

Reply:
Ah, great to know - thanks @unguiculus :) Fixing rn